images/solr/00-solr-datadir.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+set -ex
+
+# old volume looks like this
+# $corename/data/-files-
+# new volume looks like this
+# $corename/-files-
+
+for datadir in `ls -d /var/solr/*` ; do
+  corename=$(basename $datadir)
+  if [ -d ${datadir}/data ]; then
+    echo "${datadir}/data is in old format, moving to ${datadir}."
+    mv ${datadir}/data/* ${datadir}/
+    rm -Rf ${datadir}/data
+  fi
+
+  mkdir -p /opt/solr/server/solr/${corename}
```

Comment:
please comment what exactly these lines are doing and why

Reply:
updated in 31b0190b927d29